test/internal/laws/forall.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var jsc = require('jsverify');
+var R = require('ramda');
+
+//  forall :: StrMap Function -> StrMap Function
+module.exports = function forall(laws) {
+  return Object.keys(laws).reduce(function(result, name) {
+    result[name] = function() {
+      var args = Array.prototype.slice.call(arguments);
+      function pred() {
+        var pair = laws[name].apply(null, arguments);
+        var lhs = pair[0];
+        var rhs = pair[1];
+        return R.equals(lhs, rhs);
```

Comment:
That sounds reasonable to me. By the way, I'd love this to become part of a new fantasy-laws project to replace the __laws__ directory in the fantasy-land project. :)